completions/bash/podman

```diff
@@ -3363,6 +3363,82 @@ _podman_pod_inspect() {
 }
 
 
+_podman_network_create() {
+  local options_with_args="
+  -d, --driver
+      --gateway
+      --ip-range
+      --macvlan
+      --subnet
+  "
+
+  local boolean_options="
+      --disable-dns
+      --internal
+      --help
+      -h
+  "
+  _complete_ "$options_with_args" "$boolean_options"
+}
+
+_podman_network_ls() {
+  local options_with_args="
+  "
+
+  local boolean_options="
+	 --help
+	 -h
+	 -q
+	 --quiet
+  "
+  _complete_ "$options_with_args" "$boolean_options"
+}
+_podman_network_inspect() {
+  local options_with_args="
```

Comment:
`-f, --format string   Pretty-print network to JSON or using a Go template` is missing